.github/workflows/acceptance.yml

```diff
@@ -62,7 +62,7 @@ jobs:
           CI_NODE_TOTAL: 3
           CI_NODE_INDEX: ${{ matrix.instance }}
         run: |
-          JEST_TESTS=$(yarn -s jest --listTests --json) yarn test-ci --forceExit --testTimeout=10000
```

Comment:
I'm glad we're rolling this back now, can you set a reminder for yourself for next week to check if the flake changes here?

Reply:
yup, will do 👍